### PR TITLE
feat/creation of the call to action section, as instructed by Figma Designer

### PR DESCRIPTION
**Pull request by Hardsys-Andre**
I created this section according to Figma, using TailwindCSS, making the component responsive to various screen sizes, including mobile.
Below is an illustrative image of the component:
image

### CODE REVIEW

File: src/components/02-molecules/CardCallToAction.tsx
Comment (heronlancellot) [inline src/components/02-molecules/CardCallToAction.tsx:1]: hey yo @Hardsys-Andre ! GJ ! Some fixes to merge it ! 
-  [ ] Check the fonts on mobile
-  [ ] remove the Ethereum Icon & bitcoin Icon ( mobile ) like figma ;)
-  [ ] Adjust the spacing between the button and text 👍

**Review by heronlancellot — CHANGES_REQUESTED** [review]
Hey yo @Hardsys-Andre ! GJ !

**Comment by Hardsys-Andre** [comment]
@heronlancellot, The requested adjustments have been made for the mobile version, I thought the 50px size looked better in the mobile version with a screen size of 320px.

image

**Review by heronlancellot — APPROVED** [review]
hey yo @Hardsys-Andre ! 
GJ ! 🚀 🚀

**Pull request merged**